docs-v2/integrations/all/dialpad.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Dialpad
+sidebarTitle: Dialpad
+---
+
+API configuration: [`dialpad`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [Register an Application](https://airtable.com/shrju5b3Sqebrmn96)
```

Comment:
airtable mention

Reply:
@bodinsamuel, according to the provider's [docs](https://developers.dialpad.com/docs/oauth#setup), they are currently using an airtable form to enable users register for an OAuth application with Dialpad.

Reply:
I think that is wise, in case the link to the form changes in the future.